test/integration/levelDb/LevelDBTransaction.spec.js

```diff
@@ -0,0 +1,63 @@
+const level = require('level-rocksdb');
+const Transactions = require('level-transactions');
+
+const LevelDBTransaction = require('../../../lib/levelDb/LevelDBTransaction');
+
+const LevelDBTransactionIsNotStartedError = require('../../../lib/levelDb/errors/LevelDBTransactionIsNotStartedError');
+const LevelDBTransactionIsAlreadyStartedError = require('../../../lib/levelDb/errors/LevelDBTransactionIsAlreadyStartedError');
+
+describe('LevelDBTransaction', () => {
+  let db;
+  let levelDBTransaction;
+
+  beforeEach(() => {
+    db = level('./db/transaction-test', { valueEncoding: 'binary' });
```

Comment:
Why do we need db here? This test looks more like unit rather than integration.

Reply:
Because level-transactions throws "db must be a levelup instance" error.

Reply:
That's confusing. Let's use https://github.com/Level/memdown and name his variable `dbMock`?